### DISCUSSION TITLE
REQUEST: Apprise Notification API LXC

### DISCUSSION THREAD

**Discussion by TheRealFu** [tteck/Proxmox#1204] [Ideas]
Apprise has a notification service API that allows centralized configuration of notifications that can be easily triggered by a REST API (including WebHooks).  I'd love to centralize a few of my existing notifications (such as from Uptime Kuma) with a dedicated LXC.

https://github.com/caronc/apprise-api

**Comment by tristan-k** [comment]
Also comes in handy for the exisiting [changedection lxc](https://github.com/tteck/Proxmox/blob/a85593afa43095c4bd527748af7090e3e4840311/install/changedetection-install.sh).

**Comment by ilium007** [comment]
This would be a good one to have available!
